app/src/androidTest/java/ch/epfl/sweng/SDP/matchmaking/MatchmakerTest.java

```diff
@@ -48,9 +48,7 @@ public void testJoinRoom() {
         when(mockConstantsWrapper.getFirebaseUserId()).thenReturn(USER_ID);
         when(mockAccount.getUserId()).thenReturn(USER_ID);
         when(mockAccount.getUsername()).thenReturn(FAKE_USERNAME);
-        Boolean functionReturnedOK200 = Matchmaker.getInstance(mockAccount)
-                .joinRoomOther();
-        assertTrue(functionReturnedOK200);
+        Matchmaker.getInstance(mockAccount).joinRoom();
```

Comment:
Is it possible to assert something here?

Reply:
The method returns a `Task<String>` eventually I could assertNotNull(...)